tests/test_axisgrid.py

```diff
@@ -696,6 +696,19 @@ def f(grid, color):
         assert res == color
         assert g.figure.get_facecolor() == color
 
+    def test_tick_params(self):
+
+        g = ag.FacetGrid(self.df, row="a", col="b")
+        color = 'blue'
+        width = 2
+        g.tick_params(width=width, color=color)
+        for ax in g.axes.ravel():
+            for axis in ['xaxis', 'yaxis']:
+                for tick in getattr(ax, axis).get_major_ticks():
+                    assert tick.tick1line.get_color() == color
```

Comment:
Nit, using `assert mpl.colors.same_color(tick.tick1line.get_color(), color)` would make this more robust if matplotlib coerces the internal color representation to rgba in the future.